src/components/BypassLink.tsx

```diff
@@ -1,22 +1,24 @@
 import React, { ReactNode } from 'react';
-import { suomifiTheme } from 'suomifi-ui-components';
+import { suomifiDesignTokens } from 'suomifi-ui-components';
+
+import { focusOutline } from './utils/outline';
 
 const BypassLink = ({ to, children }: Props): JSX.Element => (
   <a
     href={to}
     css={[
       {
         position: 'absolute',
-        zIndex: suomifiTheme.zindexes.focus + 1,
+        zIndex: 10000,
```

Comment:
zIndexes were no longer available from the tokens and therefore a hardcoded value is now used instead.